docs/docs/faq.md

```diff
@@ -25,6 +25,9 @@ On the other hand, you can still add whatever you want as a middleware which wra
 
 As of now, No. But it should not be a big issue because the reverse-proxy which gRPC-Gateway generates usually works as an independent process and communicates with your gRPC server over TCP or a Unix domain sockets (Unix systems only).
 
+If you do insist that your gateway should be written in the same language
+as your server, there are a few projects that will help with this in [related projects](related_projects.md).
+
```

Comment:
Can we just remove this part? It doesn't feel relevant here to me.

Reply:
Done.